lib/rules/web/admin_console/3.11/goto.xyaml

```diff
@@ -5,6 +5,8 @@ goto_project_status:
   url: status/ns/<project>
 goto_resource_page_under_default_project:
   url: k8s/ns/default/<resource_url_name>
+goto_landing_page_for_empty_project:
```

Comment:
Can you explain why we need action `goto_landing_page_for_empty_project`? I assume that after user is logged in, he is expected to redirected to the correct landing page by default already, why do we need this action again?

Reply:
The landing page is different between 4.1 and 4.2, that's why automation failed.

Reply:
@yapei Dropped, let landing by logging.